bgpd/bgpd.h

```diff
@@ -47,6 +47,9 @@
 /* Default interval for IPv6 RAs when triggered by BGP unnumbered neighbor. */
 #define BGP_UNNUM_DEFAULT_RA_INTERVAL 10
 
+/* Default interval for eBGP multihop TTL. */
```

Comment:
only `Default internal` doesn't sound very good for this definition, what do you think?

Reply:
It's not "internal". It is "interval". So it should be good.

Reply:
Sorry, `interval`, but I don't think TTL is an interval, do I?